app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,274 @@
+      <div>
+      <%= render partial: "shared/messages", locals: {small_text: false} %>
+    </div>
+      <section class="usa-section padding-top-0 p0-bottom" aria-label="Metrics">
+        <div class="grid-row flex-align-end">
+          <h1 style="font-weight: bold; font-size: 40px" class="margin-bottom-3 margin-y-0 grid-col-3">
+            Metrics
+          </h1>
+          <div class="grid-col-9 text-right">
+            <a class="dm-icon-link text-no-underline text-super"
+               onclick="window.print(); return false;">
+              <i class="fas fa-print margin-right-05"></i>
+              <span>Print</span>
+            </a>
+          </div>
+        </div>
+        <div class="margin-bottom-10 grid-row flex-align-center margin-top-3">
+          <label for="metrics_duration" class="usa-label grid-col-3 x0-top">View your metrics for</label>
+          <div class="grid-col-9">
+            <select class="usa-select metrics-select x0-top" name="metrics_duration" id="metrics_duration">
+              <option id="metrics_30_day_value" value="30" <%= 'selected' if @duration.to_i == 30 %>>the last 30 days</option>
+              <option id="metrics_all_time_value" value="99999" <%= 'selected' if @duration.to_i != 30 %>>all time</option>
+            </select>
+          </div>
+        </div>
+          <div class="grid-col-8 margin-bottom-10 text-center width-full">
+          <table style="width: 80%; display: inline-table">
+            <td class="font-sans-3xs line-height-15px text-ls-1">PAGE VIEWS</td>
+            <td class="font-sans-3xs line-height-15px text-ls-1">UNIQUE VISITORS</td>
+            <td class="font-sans-3xs line-height-15px text-ls-1">BOOKMARKS</td>
+            <td class="font-sans-3xs line-height-15px text-ls-1">ADOPTIONS</td>
+            <tr>
+              <td style="font-size: 32px"><%= @page_views_for_practice_count %></td>
+              <td style="font-size: 32px"><%= @unique_visitors_for_practice_count %></td>
+              <td style="font-size: 32px"><%= @bookmarks_by_practice %></td>
+              <td style="font-size: 32px"><%= @adoptions_by_practice %></td>
+            </tr>
+          </table>
+          </div>
+        <!--  CHARTS ------------------------------------------------------------>
+
+        <div class="border bg-white padding-2 margin-bottom-10 radius-md padding-bottom-0">
+          <div style="width: 726px; height: 400px" id="chart_div"></div>
+         </div>
+
+
+<!-- ADOPTIONS ------------------------------------------------------------->
+      <div class="grid-row flex-align-center margin-bottom-3">
+        <h2 class="grid-col-6 line-height-36 font-sans-xl margin-y-0">Adoptions</h2>
+        <ul class="grid-col-6 usa-button-group usa-button-group--segmented flex-justify-end">
+          <li class="usa-button-group__item">
+            <button id="toggle_adoptions_view_30" class="usa-button dm-btn-base">Last 30 days</button>
```

Comment:
For the first button in the button group, there's an almost impossible to notice right blue border. We can git rid of this for both sets of buttons by adding to the `_practice.scss file`:

```
#toggle_adoptions_view_30, #toggle_leader_board_view_30 {
  &:before {
    border-right: $dm-color-base !important;
  }
}
```

Reply:
done.